### ISSUE TITLE
Incompatible with Node 10.x

### ISSUE THREAD

**Issue by AlexGustafsson** [jonschlinkert/github-contributors#1]
I get the following error on Node 10.2:
`Error: Cannot find module 'object.omit'`

Works fine in Node 9.x.

**Comment by doowb** [comment]
@AlexGustafsson can you provide more information? The error itself indicates that `object.omit` didn't get installed into `node_modules`. It's not used directly in this library, so seeing some code and the entire error stack would be helpful.

**Comment by AlexGustafsson** [comment]
Thank you for your fast response. I tried the package using the [NPM-suggested RunKit](https://npm.runkit.com/github-contributors). The error message I got made me think that the issue was a breaking change in later node editions. Since it worked fine for lower versions on RunKit I made the assumption that was the case.

Trying to replicate it on a local environment failed however. Turns out that RunKit doesn't seem to support usage of `object.omit` (probably a dependency of a dependency). Thank you for trying to help and sorry for any inconvenience!
